docs/ensembl-help/using-ensembl/getting-ensembl-data/tools-and-apis/graphql/getting-started.md

```diff
@@ -0,0 +1,272 @@
+---
+slug: accessing-graphql
+title: Accessing Ensembl GraphQL services
+description: How to access the GraphQL services provided by Ensembl
+related_articles:
+  - href: ./using-graphql.md
+---
+# Accessing Ensembl GraphQL services
+
+The Ensembl GraphQL services can be accessed [here](/data/graphql).
+
+## Schemas and documentation
+If you wish to interrogate the service and explore the documentation, [a GraphQL playground](/data/graphql) can be accessed via your browser.
+
+## Genomes
+
+The data within a GraphQL service is represented as a graph. The way in which the Ensembl data are logically grouped together is by genome.  Genomes are defined to be a collection of datasets and related annotation for a given assembly.  Some assemblies will have a new genome generated for each release whilst other assemblies will have genomes which span multiple releases.
+
+In many of queries, there is a parameter for `genome_id`.  This identifier (a [UUID]("https://www.itu.int/ITU-T/recommendations/rec.aspx?rec=11746&lang=en")) is used to scope queries to the data belonging to a given genome.
```

Comment:
Wrong syntax for link in markdown:

![image](https://github.com/Ensembl/ensembl-website-help-and-docs/assets/6834224/3a22eb10-3ab1-4c6d-b192-837c1e41e3ec)